docs/orchestration/concepts/cloud_hooks.md

```diff
@@ -0,0 +1,119 @@
+
+# Cloud Hooks <Badge text="Cloud"/>
```

Comment:
Can you add this page to the `/docs/.vuepress/config.js` under the orchestration concepts section?